src/utils/video-services/loadWatchLater.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+export const loadWatchLater=async(dispatchVid)=>{
+  const encodedToken=localStorage.getItem('token')
+  dispatchVid({type:"VIDEOS_LOADING"})
+    try {
+        const res=await axios({
+            method:"GET",
+            url:"/api/user/watchlater",
+            headers:{authorization:encodedToken}
+        })
+        if(res.status===200){
+            dispatchVid({type:"SET_WATCH_LATER",payload:res.data.watchlater})
```

Comment:
Features are working as intended.
UI looks great and if would look amazing if you give a different color to aside bar and Nav bar slightly color difference